main.tf

```
@@ -2556,8 +2556,8 @@ module "gatekeeper" {
   # https://github.com/open-policy-agent/gatekeeper/blob/master/charts/gatekeeper/Chart.yaml
   name             = try(var.gatekeeper.name, local.cluster_proportional_autoscaler_name)
   description      = try(var.gatekeeper.description, "A Helm chart to install Gatekeeper")
-  namespace        = try(var.gatekeeper.namespace, "gatekeeper-system")
-  create_namespace = try(var.gatekeeper.create_namespace, true)
+  namespace        = try(var.gatekeeper.namespace, "kube-system")
+  create_namespace = try(var.gatekeeper.create_namespace, false)
```

The [readme](https://github.com/open-policy-agent/gatekeeper/tree/master/charts/gatekeeper) recommends use of `gatekeeper-system`.

```suggestion
  namespace        = try(var.gatekeeper.namespace, "gatekeeper-system")
  create_namespace = try(var.gatekeeper.create_namespace, true)
```

---

Fixing in another PR.